test/functional/lib/dnslink.test.js

```diff
@@ -1,11 +1,11 @@
 'use strict'
-const { describe, it, before, beforeEach, after } = require('mocha')
-const { expect } = require('chai')
-const { URL } = require('url')
-const sinon = require('sinon')
-const createDnslinkResolver = require('../../../add-on/src/lib/dnslink')
-const { initState } = require('../../../add-on/src/lib/state')
-const { optionDefaults } = require('../../../add-on/src/lib/options')
+import { describe, it, before, beforeEach, after } from 'mocha'
+import { expect } from 'chai'
+import { URL } from 'url'
+import sinon from 'sinon'
+import createDnslinkResolver from '../../../add-on/src/lib/dnslink.js'
```

Comment:
NIT: `.js` shouldn't be necessary for these imports.

Reply:
we should standardize on esm imports and the `.js` extension is part of the filename. We should be consistent even if there is tooling that does "magic" for us. Open to hearing your thoughts @meandavejustice